### ISSUE TITLE
Top K accuracy

### ISSUE THREAD

**Issue by YuktiADY** [open-mmlab/mmaction2#2313]
Hello 

I am using STGCN method for 2d action recognition. I have trained my  model on custom dataset for around 30 epochs.
Evaluated the model,  

****Evaluating top_k_accuracy ...**

**top1_acc	0.1587 .**   
**top5_acc	0.4286****

1. Why accuracy is like 0.15xxx it means 15.8 % and 42.86%
   Why there is so much difference in top1 and top5 acc. Or it should be near by the same value ??

2. Since my dataset is small consisting of 16 actions and each performed by 5 persons each which has 60 frames each.
   So i increased my data by dividing the frames into smaller segments of seq length = 30.
   Do you know any other way through i can increase more data at  a time.

3. I want to monitor my training in tensorboard so i added this in config
   log_config = dict(interval=1, hooks=[dict(type='TextLoggerHook')])
  #log_config = dict(interval=1, hooks=[dict(type='TensorboardLoggerHook')])

But it seems if i give tensorboardloggerHook  then during training the training info of each epoch is not displayed.
Could you please how can i also use tensorboard with textloggerHook

4. Since val and test set are equal in config . Does that mean during the training the validation acurracy after epoch and evaluation of trained model will be same ?
because during training 

[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 63/63, 16.5 task/s, elapsed: 4s, ETA:     0s2023-03-24 15:46:09,387 - mmaction - INFO - Evaluating top_k_accuracy ...
2023-03-24 15:46:09,388 - mmaction - INFO - 
top1_acc	0.0952
top5_acc	0.3810
2023-03-24 15:46:09,388 - mmaction - INFO - Epoch(val) [29][63]	top1_acc: 0.0952, top5_acc: 0.3810

this is training accuracy or validation accuracy ?
